lib/task_helpers/exports/service_dialogs.rb

```diff
@@ -4,6 +4,7 @@ class ServiceDialogs
       def export(options = {})
         export_dir = options[:directory]
 
+        User.current_user = User.where(:userid => 'admin').first
```

Comment:
This seems to fix my issue.  Does it make sense to hardcode admin here?  At some point, would you want to be able to export as a less privileged user and not get everything?

Reply:
Note, if we want to use admin, we might want to call `User.super_admin` instead